app/src/main/res/values/styles.xml

```diff
@@ -647,6 +654,46 @@
         <item name="cornerSize">@dimen/top_sites_favicon_corner_size</item>
     </style>
 
+    <style name="RecentBookmarks.FaviconCard" parent="Mozac.Widgets.Favicon">
+        <item name="android:layout_width">@dimen/recent_bookmark_item_width</item>
+        <item name="android:layout_height">@dimen/recent_bookmark_item_height</item>
+        <item name="android:padding">@dimen/recent_bookmark_item_padding</item>
+        <item name="cardBackgroundColor">?mozac_widget_favicon_background_color</item>
+        <item name="cardCornerRadius">@dimen/recent_bookmark_item_card_radius</item>
+        <item name="cardElevation">@dimen/recent_bookmark_item_card_elevation</item>
+    </style>
+
+    <style name="recentBookmarkFavicon">
```

Comment:
In general, we should avoid adding these kinds of styles in future work, it's a strange precedence that exists in the code base because, the way we're using it in Fenix, doesn't seem to add any value from just setting these one-time used styles directly in the layout files.

Nothing to change here for now, just food for thought.

Reply:
That's good to know. We definitely don't want to make `styles` so huge that it becomes unusable